html-css/building-blocks/step2-instructions.md

```diff
@@ -2,13 +2,13 @@ Step 1 demonstrated how a simple component can be reused multiple times. This st
 
 When simpler components are used within more complex components, the layout of the elements are defined in the stylesheet for the complex component.
 
-## Workshop: Add a badge component to the chip component
+In this step, add a `badge` component to the `chip` component.
 
-1) <strong>Copy code into the index.html file.</strong>
+1) <strong>Copy code into the `index.html` file.</strong>
 
-Click the <strong>Copy to Editor</strong> button below to add html for a chip component to the `index.html` file.
+Click the `Copy to Editor` button below to add HTML for a chip component to the `index.html` file.
 
-<strong>Note:</strong> This is the default chip component that already has another component in it: the button! So the chip can be considered as a more complex component.
+This is the default chip component that already has another component in it: the button! This is why the chip can be considered as a more complex component.
```

Comment:
```suggestion
This is the default chip component that already has another component in it: the button. This is why the chip can be considered as a more complex component.
``` 
We didn't need the exclamation point here. We should only use that for caution or to share something exciting.